frontend-react/src/types/SubmissionDetailsTypes.ts

```diff
@@ -0,0 +1,20 @@
+export interface Destination {
```

Comment:
Are always tied to ActionDetailsResource? (e.g. if you use Destination, then the code will also be using ActionDetailsResource) Or are they more generic an used by different API calls?

If they are always related, then I'd suggest moving these declarations in the top of the ActionDetailsResource file.

Also, why is it called "ActionDetailsResource" and not "SubmissionDetailsResource"? (just curious)

Reply:
So, `Submission` is the defined term to describe a collection of Actions taken. A few things led to the naming convention, but primarily it's to keep data shapes in line front-to-back. If they make a change to their ActionDetails, it's a lot easier to know what that affects on our end rather than having to map their class names to our class names in our heads. Additionally, it looks like Sean had set that precedent by naming his `ActionResource`, so I followed suit 👍🏼